.github/workflows/deploy_ecs_relayer0_TESTNET.yml

```diff
@@ -0,0 +1,52 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: BUSL-1.1
+
+name: Deploy ECS relayer 0
+
+on:
+  release:
+    types:
+      - published
```

Comment:
@rbadillap I still don't see how this will require confirmation/approval for deployment. From my understanding, we need to have a special [environment](https://docs.github.com/en/rest/deployments/environments?apiVersion=2022-11-28) created inside this repository to set that approval is required for deployment. 
Maybe we are handling this somehow directly on AWS or some other way but just wanted to be sure that we have a manual approval for deployment set up?

Reply:
@MakMuftic that's a good question, let me elaborate what I have discovered in the AWS side plus what this piece of code does.

Technically, this trigger doesn't require an approval, because "approving" something in this context (a.k.a GitHub ecosystem) only happens on Pull Request.
So, in order to require an approval for deploying to TESTNET, you'll have to create a Pull Request changing something in the code.

Since, this can be tedious, @P1sar suggest another way where we don't necessarily require a Pull Request, but keeping the same level of awareness. 

Using [Releases](https://github.com/sygmaprotocol/sygma-relayer/releases) you can set a tag and proceed with the deploy of the branch that you'll like to allocate in the TESTNET. It doesn't require an approval, but, you are the only person who can create the release, so technically, you are approving the release of the environment.

In AWS, everything has been cloned, EXCEPT, the internal network (a.k.a VPC). The reason is because I want to monitor the traffic to consider what kind of improvements can be applied in this new environment without incurring in unnecessary charges. The switch of a brand new network won't affect the performance of the application, and will happen under your consent and approval.

About the GitHub environments, please let me do a research to evaluate what impact causes in the infrastructure level or whether the workflows needs to be changed or not.

Reply:
> Can't we use this type of setup https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments ?

This is a good suggestion, I've [created a ticket](https://github.com/sygmaprotocol/devops-backlog/issues/30) to scope this